src/main/scala/chisel3/internal/firrtl/Emitter.scala

```diff
@@ -80,8 +80,7 @@ private class Emitter(circuit: Circuit) {
         val printfArgs = Seq(e.clock.fullName(ctx), "UInt<1>(1)",
           "\"" + printf.format(fmt) + "\"") ++ args
         printfArgs mkString ("printf(", ", ", ")")
-      case e: Verification[_] => s"${e.op}(${e.clock.fullName(ctx)}, ${e.predicate.fullName(ctx)}, " +
-        s"UInt<1>(1), " + "\"" + s"${printf.format(e.message)}" + "\")"
+      case e: Verification[_] => s"${e.op}(${e.clock.fullName(ctx)}, ${e.predicate.fullName(ctx)}, UInt<1>(1), \"${printf.format(e.message)}\") : ${e.id.toNamed.name}"
```

Comment:
This fails in Scala 2.12 due to a super long running bug (finally fixed in Scala 2.13) that you can't simply escape double quotes inside of string interpolated blocks, for example: https://scastie.scala-lang.org/s4MlbfAAT4OLse6I0A6oqA

You can either do what was originally done here with the string contentions, or use triple quotes with string interpolation, eg. `s"""..."""` (and just not escape the `"` inside)

Reply:
Also, need to make sure we're doing the equivalent in the `Converter`.